MapboxNavigation/NavigationMapView.swift

```diff
@@ -498,7 +500,7 @@ open class NavigationMapView: MGLMapView {
     /**
      Removes the step arrow from the map.
      */
-    func removeArrow() {
+    public func removeArrow() {
```

Comment:
Is this intentional?

Reply:
Yeah I think this should be made public.

Reply:
Sounds reasonable since `showRoute(_:)` is public.